src/Stratis.Features.FederatedPeg/InputConsolidation/InputConsolidator.cs

```diff
@@ -172,10 +172,8 @@ public ConsolidationSignatureResult CombineSignatures(Transaction incomingPartia
         /// </remarks>
         /// <param name="amount">The consolidation amount.</param>
         /// <returns>A list of consolidation transactions.</returns>
-        public List<ConsolidationTransaction> CreateRequiredConsolidationTransactions(Money amount)
+        private List<ConsolidationTransaction> CreateRequiredConsolidationTransactions(Money amount)
         {
-            // TODO: This method doesn't need to be public.
```

Comment:
I think this TODO isn't entirely valid as its OK to have a method `public` for unit testing purposes? Unless the whole consolidation process needs to be atomic. But then the unit test or design of the class needs to be revisited.

Reply:
Hmm I don't think we should need to change method visibility for tests, so for now maybe the PR will suffice until the design can perhaps be revisited.